### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Dorcas126**
- Create a method called current_user in ApplicationController to make current user data available to all controllers.
   - It will return the first user from the database.
   - Don't worry about proper authentication at this time.  add it in the upcoming projects.
- Create forms to perform the following functions:
   - Creates a Post on behalf of the current_user (use the method that I  created in my ApplicationController).
   - Create a comment on behalf of the current_user (use the method that I created in my ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -16,4 +16,5 @@
     resources :likes, only: [:create]
   end
 end
+post '/users/:user_id/posts/:post_id/comments', to: 'comments#create', as: 'user_post_comments'
```
Comment (nahnahdev) [inline config/routes.rb]: - [ ] Great job with your implementation of the comments. However, per the project requirements, there should be a pair of routes in the `config/routes.rb` file for creating comments similar to what you did for `Posts create` and `Posts New`. Currently, you only have the `Post` route for comments in your project :+1: 
- Example
   ```ruby
      POST   /posts/:post_id/comments(.:format)     comments#create
    GET   /posts/:post_id/comments/new(.:format)     comments#new
   ```
- [ ] After doing this, you should ensure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments. :+1:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Dorcas126  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- Linters well setup and passing :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- ApplicationController contains `#current_user` method :heavy_check_mark: 
- Most required routes present :heavy_check_mark: 
- Most required forms present :heavy_check_mark: 
- User is able to create a post and add a like to posts :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Dorcas126 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost got it


 ## Highlights
Nice design :+1: 
Great effort in making requested changes :+1: 

 ## Required Changes ♻️

- [ ] Nice job so far. However, as requested by the previous reviewer, kindly ensure that there is a separate form in the views/comments/new.html.erb file that accepts input for creating comments. Remember, this is part of the project requirements :pray: 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Dorcas126👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with forms. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**